astropy/utils/tests/test_data.py

```diff
@@ -2112,3 +2113,16 @@ def test_empty_url(temp_cache, valid_urls):
     u, c = next(valid_urls)
     download_file('file://', cache=True, sources=[u])
     assert not is_url_in_cache('file:///')
+
+
+@pytest.mark.parametrize(
+    ('s', 'ans'),
+    [('http://googlecom', True),
+     ('https://google.com', True),
+     ('ftp://google.com', True),
+     ('sftp://google.com', True),
+     ('ssh://google.com', True),
+     ('file:///c:/path/to/the%20file.txt', True),
+     ('google.com', False)])
+def test_string_is_url_check(s, ans):
+    assert is_url(s) is ans
```

Comment:
Do we really want to require that the specific object True is required rather than something that evaluates as True?

Reply:
I only use `is` to avoid `if` logic when `ans` is expected to be `False`.